src/python/grapl-template-generator/grapl_template_generator/grapl-templates/grapl-python-service-template/{{cookiecutter.project_slug}}/BUILD

```diff
@@ -0,0 +1,6 @@
+python_awslambda(
+    name="lambda",
+    runtime="python3.7",
+    handler="{{cookiecutter.pkg_name}}.py/main.py:{{cookiecutter.lambda_handler}}"
+)
+
```

Comment:
Yep, though I thought I'd keep it around in case we wanted a python http service in the future (de-lambdaifying isn't the worst thing in the world). However, I have signoff from colin to just claw this all out. RUST FOREVER

Reply:
We should remove all of the python stuff